build-tools/helm/spark-kubernetes-operator/crds/sparkapplications.org.apache.spark-v1.yml

```diff
@@ -0,0 +1,6947 @@
+# Generated by Fabric8 CRDGenerator, manual edits might get overwritten!
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: sparkapplications.org.apache.spark
+spec:
+  group: org.apache.spark
+  names:
+    kind: SparkApplication
+    plural: sparkapplications
+    shortNames:
+      - sparkapp
+    singular: sparkapplication
+  scope: Namespaced
+  versions:
+    - name: v1alpha1
+      schema:
+        openAPIV3Schema:
+          properties:
+            spec:
+              properties:
+                mainClass:
+                  type: string
+                runtimeVersions:
+                  properties:
+                    sparkVersion:
+                      enum:
+                        - v3_5_1
+                        - v3_5_0
+                        - v3_4_2
+                        - v3_4_1
+                        - v3_4_0
+                        - v3_3_3
+                        - v3_3_2
+                        - v3_3_1
+                        - v3_3_0
+                        - v3_2_0
```

Comment:
Please follow Apache Spark community release policy by removing all EOL versions.

Reply:
removed old versions except 3.4.1 - once spark-docker releases 3.4.2 I'll replace that asap.